src/app/app.component.css

```diff
@@ -2,4 +2,169 @@
     min-height: 100vh;
     display: flex;
     flex-direction: column;
+
+    --zm-grey: #cdc7c2;
+    --zm-green: #157347;
+
+    --zm-header-light-bg: #ececec;
+    --zm-header-dark-bg: var(--bs-dark);
+
+    --zm-navbar-light-color: var(--bs-body-color);
+    --zm-navbar-light-bg: #ffffff;
+    --zm-navbar-link-light-color: rgba(0, 0, 0, 0.55);
+    --zm-navbar-link-light-color-hover: rgba(0, 0, 0, 0.7);
+    --zm-navbar-link-active-light-color: rgba(0, 0, 0, 0.9);
+
+    --zm-navbar-dark-color: var(--bs-light);
+    --zm-navbar-dark-bg: var(--bs-dark);
+    --zm-navbar-active-dark-color: #ffffff;
+    --zm-navbar-link-dark-color: rgba(255, 255, 255, 0.55);
+    --zm-navbar-link-dark-color-hover: rgba(255, 255, 255, 0.75);
+    --zm-navbar-link-active-dark-color: #ffffff;
+
+    --zm-footer-light-color: var(--bs-body-color);
+    --zm-footer-light-bg: #ffffff;
+    --zm-footer-dark-color: var(--bs-light);
+    --zm-footer-dark-bg: var(--bs-dark);
+
+    --zm-body-light-bg: #ffffff;
+    --zm-body-light-color: var(--bs-body-color);
+    --zm-body-dark-bg: #2C3034;
+    --zm-body-dark-color: var(--bs-light);
+
+    --zm-link-light-color: var(--bs-link-color);
+    --zm-link-light-color-hover: var(--bs-link-hover-color);
+    --zm-link-dark-color: #5a9dff;
+    --zm-link-dark-color-hover: #8ea5ff;
+
+    --zm-form-btn-bg-1: #ff5a5f;
+    --zm-form-btn-bg-2: #ff3c3a;
```

Comment:
Maybe `--zm-form-btn-bg` and `--zm-form-btn-bg-hover` (as you did for other) are more explicit.